src/Bart/Jenkins/Job.php

```diff
@@ -9,57 +8,72 @@
  */
 class Job
 {
-	private $base_job_url;
-	private $default_params = array();
-	private $my_build_id;
+	private $defaultParams = [];
+	private $myBuildId;
 	private $metadata;
 	/** @var \Logger */
 	private $logger;
 
+	/** @var Connection $connection */
+	private $connection;
+	/** @var string $baseApiPath */
+	private $baseApiPath;
+
 	/**
-	 * Load metadata about a project
-	 */
-	public function __construct($domain, $job_name)
+	 * Job constructor. Loads metadata about a project.
+	 * @param Connection $connection
+	 * @param array $projects This parameter specifies the location of the Jenkins Job.
```

Comment:
I keep getting thrown off by this. Why is it an array? That doesn't really make sense to user of this code. I would suggest accepting the "path" to the job and then internally managing it as an array if that's easiest.